### DISCUSSION TITLE
Display reverse DNS result as alias

### DISCUSSION THREAD

**Discussion by MyPhonesAndrew** [sipcapture/homer#538] [Ideas]
Is there a way the alias field can be automatically populated by performing a reverse DNS lookup for the IP address?

**Comment by lmangani** [comment]
Hello @myp-andrew-kirwin I'm afraid this feature is only available in HEPIC currently. Would you like to PR or sponsor this feature?

**Reply by MyPhonesAndrew** [reply]
Hi @lmangani What does Sponsorship look like exactly?
